apps/meteor/tests/e2e/omnichannel/omnichannel-livechat.spec.ts

```diff
@@ -93,6 +93,57 @@ test.describe.serial('OC - Livechat', () => {
 	});
 });
 
+test.describe.serial('OC - Livechat - Visitors closing the room is disabled', () => {
+	let poLiveChat: OmnichannelLiveChat;
+	let poHomeOmnichannel: HomeOmnichannel;
+
+	test.beforeAll(async ({ api }) => {
+		await api.post('/livechat/users/agent', { username: 'user1' });
+	});
+
+	test.beforeAll(async ({ browser, api }) => {
+		const { page: livechatPage } = await createAuxContext(browser, Users.user1, '/livechat', false);
+
+		poLiveChat = new OmnichannelLiveChat(livechatPage, api);
+	});
+
+	test.beforeAll(async ({ browser, api }) => {
+		await api.post('/settings/Omnichannel_allow_visitors_to_close_conversation', { value: false });
```

Comment:
I'd suggest using our `setSettingValueById` helper instead of calling the endpoint directly